Static-Pages/checkout/checkout.css

```diff
@@ -0,0 +1,73 @@
+@import '../../styles.css';
+@import '../cartpage/cart.css';
+
+/* Checkout container */
+.chk--container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+/* Address styling */
+.card {
+  max-width: none;
+  max-height: none;
+  width: 70%;
+  height: auto;
+  margin: 1rem auto;
+}
+.fields {
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  flex-direction: column;
+  margin: 0.5rem 0;
+}
+.lbl {
+  display: inline-block;
+  flex: 1;
+}
+.ipt {
```

Comment:
This type of class name  is bit confusing, try to use more understandable class name

Reply:
ok I'll work on that